aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -170,15 +170,12 @@ std::tuple<Tensor, Tensor, Tensor> nested_layer_norm(
       std::nullopt /* device */,
       std::nullopt /* pin_memory */,
       at::MemoryFormat::Contiguous);
-  auto options = input_buffer.options();
-  if (input_buffer.is_cuda()) {
-    auto acc_type = at::toAccumulateType(input_buffer.scalar_type(), true);
-    options = options.dtype(acc_type);
-  }
+  auto options = input_buffer.options().dtype(
```

Comment:
Nit: This behavior is modified if the input_buffer's device is a CPU, but it seems acceptable to implement this change.

Reply:
@swolchok Could you help review this change?

Reply:
I'm not the right person to review this. Haven't touched this stuff in years. That said, I generally wouldn't attempt to change behavior on CPU like this without a good reason, because people are likely to come out of the woodwork later saying they depended on the old behavior.

Reply:
Thank you for your suggestion. We will retain the current CPU behavior as it is.